packages/core/src/components/Carousel/Carousel.js

```diff
@@ -1,48 +1,77 @@
+import 'mutationobserver-shim';
+
 import PropTypes from 'prop-types';
 import React, { useState, useEffect } from 'react';
 import cx from 'classnames';
 import NukaCarousel from 'nuka-carousel';
-import shortid from 'short-id';
+import nanoid from 'nanoid';
 import Icon from '../Icon/Icon';
-import SectionHeader from '../Type/SectionHeader/SectionHeader';
+import BtnContainer from '../BtnContainer/BtnContainer';
+import ScreenReadable from '../ScreenReadable/ScreenReadable';
 
 import css from './Carousel.css';
 
 const Carousel = props => {
+  const id = nanoid();
   const [slideIndex, setSlideIndex] = useState(0);
 
-  const renderDesktopControls = () => {
+  const renderTopRightControls = () => {
+    if (!props.useTopRightControls) return null;
```

Comment:
I think it reads better to put this condition outside of the function and only call the function when necessary:
```jsx
{props.useTopRightControls && renderTopRightControls()}
```

Reply:
yep agreed. Reads much better